src/components/Content.js

```diff
@@ -39,6 +40,12 @@ class Content extends Component {
             exact
             component={MyArticles}
           />
+          <ProtectedRoute
+            path="/articles/:articleId"
+            component={Article}
+            isAuthenticated={isAuthenticated}
+            exact
+          />
```

Comment:
Add one for /:articleId without auth as well

Reply:
Will do